docs/guides/getting-started-java-spring.md

```diff
@@ -37,13 +37,9 @@ Before getting started, ensure you have the following in your local environment:
 
 - Access to your preferred code editor or IDE
 - [OpenJDK 21+](https://openjdk.org/install/) and [Maven 3](https://maven.apache.org/index.html)
+  - After installing OpenJDK, ensure `JAVA_HOME` is set by running `java -version` in a **new** terminal. If no version of Java is found, follow your chosen installation's instructions for setting `JAVA_HOME` before continuing.
 - [Camunda 8 Desktop Modeler](/components/modeler/desktop-modeler/index.md) and [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md)
-
-:::note
-After installing OpenJDK, ensure `JAVA_HOME` is set by running `java -version` in a **new** terminal.
-
-If no version of Java is found, follow your chosen installation's instructions for setting `JAVA_HOME` before continuing.
-:::
+  - Ensure you are running Camunda 8 Run as a Java application. The Docker Compose distribution is not yet supported.
```

Comment:
Since the "default" way to onboard is through C8Run, we can probably just keep the first sentence regardless of your test, but I am curious if the Docker Compose method introduces friction.